locale/merge_csl_locales

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import sys
+import urllib.request
+import shutil
+import xml.etree.ElementTree as ET
+import unicodedata
+
+LOCALES_GIT_REF = "bd8d2dbc85713b192d426fb02749475df30f0d2c"
+
+
+def main():
+    use_cache = True
+
+    script_dir = os.path.dirname(os.path.realpath(__file__))
+    if len(sys.argv) < 2 or not os.path.isdir(sys.argv[1]):
+        sys.stderr.write(
+            "Usage: {0} path/to/zotero/source\n".format(os.path.basename(sys.argv[0]))
+        )
+        return 1
+
+    source_dir = sys.argv[1]
+    schema_dir = os.path.join(source_dir, "resource", "schema")
+
+    csl_locale_base = "https://raw.githubusercontent.com/citation-style-language/locales/{ref}/locales-{lang}.xml"
+
+    # Codes for the language packs that we want to grab
+    language_packs = [
+        "af-ZA",
+        "ar",
+        "bg-BG",
+        "ca-AD",
+        "cs-CZ",
+        "cy-GB",
+        "da-DK",
+        "de-AT",
+        "de-CH",
+        "de-DE",
+        "el-GR",
+        "en-GB",
+        "en-US",
+        "es-CL",
+        "es-ES",
+        "es-MX",
+        "et-EE",
+        "eu",
+        "fa-IR",
+        "fi-FI",
+        "fr-CA",
+        "fr-FR",
+        "he-IL",
+        "hi-IN",
+        "hr-HR",
+        "hu-HU",
+        "id-ID",
+        "is-IS",
+        "it-IT",
+        "ja-JP",
+        "km-KH",
+        "ko-KR",
+        "la",
+        "lt-LT",
+        "lv-LV",
+        "mn-MN",
+        "nb-NO",
+        "nl-NL",
+        "nn-NO",
+        "pl-PL",
+        "pt-BR",
+        "pt-PT",
+        "ro-RO",
+        "ru-RU",
+        "sk-SK",
+        "sl-SI",
+        "sr-RS",
+        "sv-SE",
+        "th-TH",
+        "tr-TR",
+        "uk-UA",
+        "vi-VN",
+        "zh-CN",
+        "zh-TW",
+    ]
+
+    number_formats = {}
+
+    for lang in language_packs:
+        url = csl_locale_base.format(ref=LOCALES_GIT_REF, lang=lang)
+        pack_dir = os.path.join(script_dir, "csl_packs", lang)
```

Comment:
`csl_packs` → `csl_locales`. These aren't really "language packs" like the other script is dealing with.

But also, I don't think we need to cache these at all by default. That's mostly useful for development, so if anything it should be a `--cache` flag that controls both saving and using (or this can just be a boolean that's hard-coded to False). Again, we'll want to just re-run this very occasionally to process updated locales, and it shouldn't be necessary to delete the cache folder manually.

Reply:
>That's mostly useful for development

Yeah, I really just added caching so that I could quickly iterate when working on the text processing part, and it didn't occur to me that it pretty much defeats the point. I'll remove it.